Floofbot/Modules/TagCommands.cs

```diff
@@ -105,34 +105,39 @@ private bool UserHasTagUpdatePermissions(IGuildUser user)
                 return;
             }
 
-            if (!string.IsNullOrEmpty(tag) && !string.IsNullOrEmpty(content))
+            if (!string.IsNullOrEmpty(tagName) && !string.IsNullOrEmpty(content))
             {
-                Regex rgx = new Regex("[^a-zA-Z0-9 -]");
-                tag = rgx.Replace(tag, "").ToLower();
-                string tagName = $"{tag.ToString()}:{Context.Guild.Id}";
+                Regex rgx = new Regex("[^a-zA-Z0-9-]");
+                string processedTagName = rgx.Replace(tagName, "").ToLower();
+                if (string.IsNullOrEmpty(processedTagName))
+                {
+                    await SendEmbed(CreateDescriptionEmbed($"💾 Invalid Tag name. Tag must contain characters within [A-Za-z0-9-]."));
```

Comment:
I'm not sure because of githubs way of displaying it, but the line seems a bit long and perhaps breaking it up is good

Reply:
Fixed